RELEASE/data/autoscend_familiars.txt

```diff
@@ -27,7 +27,7 @@ drop	0	Fist Turkey	prop:_turkeyBooze<5
 # drops 1 per combat with chance of 2nd if wearing familiar specific equip
 drop	1	Puck Man	item:Yellow Pixel<20
 drop	2	Ms. Puck Man	item:Yellow Pixel<20
-# 1st wax drop per run only takes 5 combats, afterwards 30. makes a single size 2 density 4.25 food or drink
+# 1st wax drop per run only takes 5 combats, afterwards 30. makes a single size 2 density 4.25 food or drink 
```

Comment:
?

Reply:
I needed to re-build our data files due to to making change to setting. when doing so this whitespace change came up too. suggesting that someone changed the whitespace in the datafile, commit it, but did not commit the associated changed built file in /data/ directory.